test/verifiedTests/testFVA/testFVA.m

```diff
@@ -48,36 +49,83 @@
             parpool(2);
         end
 
-        % launch the flux variability analysis
-        [minFluxT, maxFluxT] = fluxVariability(model, 90);
-
-        rxnNames = {'PGI', 'PFK', 'FBP', 'FBA', 'TPI', 'GAPD', 'PGK', 'PGM', 'ENO', 'PYK', 'PPS', 'G6PDH2r', 'PGL', 'GND', 'RPI', 'RPE', 'TKT1', 'TKT2', 'TALA'};
-
-        % retrieve the IDs of each reaction
-        rxnID = findRxnIDs(model, rxnNames);
-
-        % check if each flux value corresponds to a pre-calculated value
-        for i = 1:size(rxnID)
-            % test the components of the minFlux and maxFlux vectors
-            assert(minFlux(i) - tol <= minFluxT(i))
-            assert(minFluxT(i) <= minFlux(i) + tol)
-
-            assert(maxFlux(i) - tol <= maxFluxT(i))
-            assert(maxFluxT(i) <= maxFlux(i) + tol)
-
-            maxMinusMin = maxFlux(i) - minFlux(i);
-            maxTMinusMinT = maxFluxT(i) - minFluxT(i);
-            assert(maxMinusMin - tol <= maxTMinusMinT)
-            assert(maxTMinusMinT <= maxMinusMin + tol)
-
-            % print the labels
-            printLabeledData(model.rxns(i), [minFlux(i) maxFlux(i) maxFlux(i)-minFlux(i)], true, 3);
+        for m={'parallel','sequential'}
```

Comment:
parallel has to be first, because sequential turns off the parallel pool which might be needed for the consecutive tests.